spec/src/main/asciidoc/config.asciidoc

```diff
@@ -56,17 +56,34 @@ has a class with the same name.
 The result is that we should have exclusive class names in MicroProfile for classes that might need configuration override in the process environment. 
 The 'first' specification that owns a class name should own that name exclusively. 
 
-Where user class names appear, these are expected to be package qualified.
+Where application class names appear, these are expected to be package qualified.
 
 [[overloadedMethods]]
 ==== Properties That Relate To Overloaded Methods
 
 There are places in the configuration where a method name is used. 
 Accounting for methods with the same name but different signatures
-can be done using indexing. 
-The "1" in the Property name below refers to the occurrence of the 'method' named 
+does not have a recommended approach yet.
+
+[[parameterAnnotations]]
+==== Properties That Relate To Annotated Parameters in Application Methods
+
+Parameter names are not guaranteed to be available during annotation processing.
+Due to this, it can be problematic to rely on parameter names for annotations that
+relate to application methods.
+An annotated application method parameter can be referred to using a positional index, starting at '1'. 
+The "1" in the Property name below refers to the first parameter
 ----
-`org.eclipse.microprofile.example.ExampleBean/setCompletableFuture/1/ManagedExecutorCon fig/maxAsync=5`
+   org.eclipse.microprofile.example.ExampleBean/setCompletableFuture/1/ManagedExecutor/maxAsync=5
+----
+
+[[fieldAnnotations]]
+==== Properties That Relate To Annotated Fields
+
+Is is possible to override attributes of annotated fields. In this case the following
+pattern is recommended.
+----
+   <fully qualified class name>/<field name>/<spec-defined annotation name>/<annotation attribute name>
 ----
 
```

Comment:
Given that we have explicit sections for fieldAnnotations and parameterAnnotations (thanks for adding these!), it might be nice to have an explicit section for methodAnnotations. There are some good examples currently in use by MP Fault Tolerance spec here,
https://github.com/eclipse/microprofile-fault-tolerance/blob/master/spec/src/main/asciidoc/configuration.asciidoc
such as,
```
com.acme.test.MyClient/serviceA/Retry/maxDuration=3000
```

Reply:
Added.